packages/core/src/analytics/integrations/__tests__/Omnitracking.test.js

```diff
@@ -372,7 +373,96 @@ describe('Omnitracking', () => {
           expect(mockLoggerWarn).not.toHaveBeenCalled();
         });
       });
+
+      describe('Event Tracking', () => {
+        describe('Interact Content', () => {
+          it('should warn if an interact content does not have required parameters', async () => {
+            const data = generateTrackMockData({
+              event: eventTypes.INTERACT_CONTENT,
+              properties: {
+                interactionType: undefined,
+              },
+            });
+            await omnitracking.track(data);
+
+            expect(mockLoggerWarn).toHaveBeenCalledWith(
+              expect.stringContaining(
+                'requires at least properties contentType',
+              ),
+            );
+          });
+
+          it('should track scroll event', async () => {
+            const data = generateTrackMockData({
+              event: eventTypes.INTERACT_CONTENT,
+              properties: {
+                interactionType: interactionTypes.SCROLL,
+                target: document.body,
+                percentageScrolled: 50,
+              },
+            });
+            await omnitracking.track(data);
+
+            expect(postTrackingsSpy).toHaveBeenCalledWith(
+              expect.objectContaining({
+                parameters: expect.objectContaining({
+                  tid: 668,
+                }),
+              }),
+            );
+          });
+        });
+
+        it('should not track any event', async () => {
+          const data = generateTrackMockData({
+            event: eventTypes.INTERACT_CONTENT,
+            properties: {
+              interactionType: interactionTypes.SCROLL,
+              target: undefined,
+              percentageScrolled: 50,
+            },
+          });
+          await omnitracking.track(data);
+
+          expect(postTrackingsSpy).toHaveBeenCalledTimes(0);
+        });
+
+        it('should track a default interact content event', async () => {
+          const data = generateTrackMockData({
+            event: eventTypes.INTERACT_CONTENT,
+            properties: {
+              interactionType: interactionTypes.CLICK,
+              contentType: 'logo',
+              id: 'home_logo',
+            },
+          });
+          await omnitracking.track(data);
+
+          expect(postTrackingsSpy).toHaveBeenCalledWith(
+            expect.objectContaining({
+              parameters: expect.objectContaining({
+                tid: 2882,
```

Comment:
Missing the remaining parameters: `contentType`, `interactionType`, `val` and `actionArea`. Use an event that will generate an omnitracking payload that contain all of those properties, to make sure the test cover it.

Reply:
This parameters will be tested with iteration of each event (using fixtures data), that should be equal than a snapshot with these fields.

Reply:
Yes, I was looking at the latest commit only which did not have changes on the snapshots but I see that it was in a previous commit relative to a previous PR.